gno.land/pkg/integration/testscript_gnoland.go

```diff
@@ -291,7 +291,9 @@ func gnolandCmd(t *testing.T, nodesManager *NodesManager, gnoRootDir string) fun
 			if *lockTransfer {
 				genesis.Bank.Params.RestrictedDenoms = []string{"ugnot"}
 			}
+
 			genesis.VM.RealmParams = append(genesis.VM.RealmParams, tsGenesis.VM.RealmParams...)
+			genesis.Auth.Params.FeeCollector = tsGenesis.Auth.Params.FeeCollector
```

Comment:
I should be able to set all the parameter values here, but the genesis toml file should then contain all the default values for the `Auth` module.

Reply:
the more default values you can set in the genesis_params.toml file (with comments), the better. it's doc.

Reply:
That makes sense. I like having this `genesis_params` as the source of truth for `gno.land` params. Should we apply them with a right merge? It would be better to have the file override the default values set in `params.go` files instead of having the file replace all the values, even empty one.